frontend/src/view/Aplose_presentation/aplosestyles.css

```diff
@@ -0,0 +1,195 @@
+body, html {
```

Comment:
Avoid writing html affecting global html in a page or component css.
In fact this css is undirectly imported right from the `App.tsx` file so it will be aplied to all the Aplose website